src/OpenLoco/Vehicles/VehicleHead.cpp

```diff
@@ -1236,7 +1236,7 @@ namespace OpenLoco::Vehicles
                 if (var_0C & Flags0C::manualControl)
                 {
                     auto* vehType2 = train.veh2;
-                    if (vehType2->var_36 != var_36 || vehType2->var_2E != var_2E)
+                    if (vehType2->routingHandle != routingHandle || vehType2->subPosition != subPosition)
```

Comment:
vehicle 2 likely represents the destination that the vehicle is trying to get to.